engines/agos/input_pn.cpp

```diff
@@ -34,6 +34,13 @@ void AGOSEngine_PN::handleKeyboard() {
 	if (!_inputReady)
 		return;
 
+	if (_keymapEnabled == true) {
+		Common::Keymapper *keymapper = AGOSEngine::getEventManager()->getKeymapper();
+		keymapper->getKeymap("agos-main")->setEnabled(false);
+		keymapper->getKeymap("game-shortcuts")->setEnabled(false);
+		_keymapEnabled == false;
```

Comment:
```suggestion
		_keymapEnabled = false;
```

It should also be turned back on when text input is finished.

Reply:
if I turn the keymaps back on at the end of the function right after text input, it does not register the keys that are mapped already for some reason